nix-darwin/modules/buildkite-agent.nix

```diff
@@ -27,6 +27,12 @@ in {
       else
         (>&2 echo "$creds doesn't exist. The build is going to fail.")
       fi
+      NIX_PROFILE_BINPATH=/nix/var/nix/profiles/default/bin
```

Comment:
is `pre-command` the best hook for this? I thought it was there was a global environment hook.

Reply:
There is an environment hook that's used for secrets. I guess it wouldn't hurt to move it there. Then any checkout/artifact parts of a job would also have access to nix.